### ISSUE TITLE
Color of svg changed.

### ISSUE THREAD

**Issue by hellomrbigshot** [vbenjs/vite-plugin-svg-icons#45]
I have a svg file called menu-more.svg
```
<svg width="50" height="50" viewBox="0 0 50 50" fill="none" xmlns="http://www.w3.org/2000/svg">
<rect width="50" height="50" fill="#353A58"/>
<path d="M17 18H33" stroke="#F2F4FA" stroke-width="2" stroke-linecap="round"/>
<path d="M17 25H33" stroke="#F2F4FA" stroke-width="2" stroke-linecap="round"/>
<path d="M17 32H33" stroke="#F2F4FA" stroke-width="2" stroke-linecap="round"/>
</svg>
```

It used to be like this
<img width="49" alt="menu-more-light" src="https://user-images.githubusercontent.com/16463481/152968294-9a5601e8-ca42-4257-976c-4467d45febe3.png">

But when i import this with the plugin, it shows like this
<img width="46" alt="menu-more-dark" src="https://user-images.githubusercontent.com/16463481/152969529-20d2a03c-33fc-40e5-81be-fc8286986ce9.png">

Color of path changed. How it happens.

**Comment by baiye0** [comment]
Lastest version stroke is replaced.
![image](https://user-images.githubusercontent.com/46374906/153178601-8a8bdeff-a39a-4e46-8ec2-840c466ab975.png)
The new version is not compatible.

**Comment by ZuulAndRibbon** [comment]
> Lastest version stroke is replaced. ![image](https://user-images.githubusercontent.com/46374906/153178601-8a8bdeff-a39a-4e46-8ec2-840c466ab975.png) The new version is not compatible.

I have the same problem.How to change the color

**Comment by michaelhue** [comment]
While this behavior should probably be opt-in, it's actually quite useful (at least for all my use cases).
The [`currentColor`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword) keyword represents the element's `color` CSS property.

In other words: in order to change the icon color,  change the `color` property of the icon element or any parent element.

**Comment by peterdzl** [comment]
I agree this should be an opt-in - at present there is no way to leave the SVG stroke colour untouched. Our use case is that we have multiple colours and we define currentColor in the source SVG where we want to make use of CSS overrides.

**Comment by nelsonjd** [comment]
Thank you @digitalacorn. I would like to opt-in to this stroke replacing as well.  Can we get this merged soon?

**Comment by inhyuck** [comment]
There are other libraries as well.
https://github.com/Jevon617/unplugin-svg-component
https://github.com/Jevon617/unplugin-svg-component?tab=readme-ov-file#plugin-configuration `preserveColor` option
